content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+    Additional setting for volume
+
+    *default:* {}
+  * `name`
+    Optional disk name. By default it will be disk id.
+
+    *type:* string
+    *default:*
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the Google Cloud Platform API.
+
+    *default:* {}
+  * `image`
+    The image of the Volume.
+
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists or if Cloudify should create the resource.
+
+    *type:* boolean
+    *default:* False
+  * `size`
+    Size of the Volume in GB.
+
+    *type:* integer
+    *default:* 10
+
+
+
+
+
+
+
+# Relationships
+
+## cloudify.gcp.relationships.contained_in_compute
+**Derived From:** cloudify.relationships.contained_in
+
+
+## cloudify.gcp.relationships.contained_in_network
+**Derived From:** cloudify.relationships.contained_in
+
+
+## cloudify.gcp.relationships.dns_record_connected_to_instance
+**Derived From:** cloudify.relationships.connected_to
+
+
+## cloudify.gcp.relationships.dns_record_connected_to_ip
+**Derived From:** cloudify.relationships.connected_to
+
+
+## cloudify.gcp.relationships.dns_record_contained_in_zone
+**Derived From:** cloudify.relationships.contained_in
+
+
+## cloudify.gcp.relationships.file_system_contained_in_compute
+**Derived From:** cloudify.relationships.contained_in
+
+
+## cloudify.gcp.relationships.forwarding_rule_connected_to_target_proxy
+**Derived From:** cloudify.relationships.connected_to
+
+
+## cloudify.gcp.relationships.instance_connected_to_disk
+**Derived From:** cloudify.relationships.connected_to
+
+
+## cloudify.gcp.relationships.instance_connected_to_instance_group
+**Derived From:** cloudify.relationships.connected_to
+
+
+## cloudify.gcp.relationships.instance_connected_to_ip
+**Derived From:** cloudify.relationships.connected_to
+
+
+## cloudify.gcp.relationships.instance_connected_to_keypair
+**Derived From:** cloudify.relationships.connected_to
+
+
+## cloudify.gcp.relationships.instance_connected_to_security_group
+**Derived From:** cloudify.relationships.connected_to
+
+
+## cloudify.gcp.relationships.instance_contained_in_network
+**Derived From:** cloudify.relationships.contained_in
+
+
+## cloudify.gcp.relationships.uses_as_backend
+**Derived From:** cloudify.relationships.connected_to
+
+
+
+
+# Account Information
+
+The plugin needs access to your GCP auth credentials (via the [`gcp_config`](#common-properties) parameter) in order to operate (but see below about use within a manager).
+
+
+# Using a Manager
+
+## Bootstrapping
+Use the [simple manager blueprint]{{< relref "/manager/bootstrapping" >}} to bootstrap a manager on a CentOS 7 Instance with at least 4GB of RAM.
+
+## `gcp_config`
+If you don't want to provide the `gcp_config` dictionary to every node in your blueprints, you can provide it, as `json`, at `/etc/cloudify/gcp_plugin/gcp_config`
+
+
+## Networks
+
+Instances in GCP are not able to communicate internally with instances in a different network.
```

Comment:
Instances in GCP are ~~not~~ **un**able to communicate internally with instances in a different network.
 ~~This means that i~~ **I**f you want to run Cloudify agents on your nodes they must be in the same network as ~~the m~~ **Cloudify M**anager.